moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -164,7 +164,7 @@ MotionPlanningFrame::MotionPlanningFrame(MotionPlanningDisplay* pdisplay, rviz::
     }
     catch (std::exception& ex)
     {
-      // ROS_ERROR("Object recognition action: %s", ex.what());
+      // RCLCPP_ERROR("Object recognition action: %s", ex.what());
```

Comment:
Enable this again, use logger name as specified https://github.com/AcutronicRobotics/moveit2/pull/101/files#r292774419